modules/qna/src/views/full/Components/QnA.tsx

```diff
@@ -0,0 +1,138 @@
+import { Button, Icon } from '@blueprintjs/core'
+import { confirmDialog, lang, MoreOptions } from 'botpress/shared'
+import cx from 'classnames'
+import _uniqueId from 'lodash/uniqueId'
+import React, { FC, Fragment, useEffect, useRef, useState } from 'react'
+
+import { MoreOptionsItems } from '../../../../../../src/bp/ui-shared/src/MoreOptions/typings'
+import style from '../style.scss'
+
+import TextareaList from './TextareaList'
+
+interface Props {
+  question: any
+  contentLang: string
+  updateQnA: (questions: any) => void
```

Comment:
Please avoid any as much as possible. Here I think it should be QnaItem (defined in qna.d.ts, which could be renamed to typings.d.ts to be consistent)